### PR TITLE
Attestation

### PR DESCRIPTION
**Pull request by abhaydaga**

### CODE REVIEW

File: 09-AzureDevOps(VSTS)-Security/Readme.md [hunk 1]
```diff
@@ -211,6 +212,172 @@ Step 1,2 & 3 needs to be repeated to add “__Project Component Security Scan Su
...
+|Attestation Option|Description|
+|------------------|-----------|
+|NotAttested|Attest only those controls which have not been attested yet.|
+|AlreadyAttested|Attest those controls which have past attestations. To re-attest or clear attestation.|
+|All|Attest all controls which can be attested (including those that have past attestations).|
+|None|N/A.|
+
+For example, to attest organization controls, run the command below:
+```PowerShell  
+$subscriptionId = <Your SubscriptionId>
```
Comment (arvindsingh-cloud) [inline 09-AzureDevOps(VSTS)-Security/Readme.md:270]: Subscriptionid?

File: 09-AzureDevOps(VSTS)-Security/Readme.md [hunk 2]
```diff
@@ -211,6 +212,172 @@ Step 1,2 & 3 needs to be repeated to add “__Project Component Security Scan Su
...
+Get-AzSKAzureDevOpsSecurityStatus -OrganizationName $orgName -ControlsToAttest NotAttested -ResourceTypeName Organization
```
Comment (arvindsingh-cloud) [inline 09-AzureDevOps(VSTS)-Security/Readme.md:271]: Add cmd for project also
Get-AzSKAzureDevOpsSecurityStatus -OrganizationName $orgName -ProjectNames $prgNames -ControlsToAttest NotAttested -ResourceTypeName Project

File: 09-AzureDevOps(VSTS)-Security/Readme.md [hunk 3]
```diff
@@ -211,6 +212,172 @@ Step 1,2 & 3 needs to be repeated to add “__Project Component Security Scan Su
...
+|Passed |Fully automated control. Azure DevOps artifact configuration meeting the control requirement|
+|Verify |Semi-automated control. It would emit the required data in the log files which can be validated by the user/auditor.|
+|Failed |Fully automated control. Azure DevOps artifact configuration not meeting the control requirement|
+|Error |Automated control. Currently failing due to some exception. User needs to validate manually|
+|Manual |No automation as of now. User needs to validate manually|
+|Exception |Risk acknowledged. The 'WillNotFix' option was chosen as attestation choice/status. |
+
+[Back to top...](Readme.md#contents)
+### Permissions required for attesting controls:
+Attestation is currently supported only for organization and project controls with admin privileges on organization and project, respectively.
```
Comment (arvindsingh-cloud) [inline 09-AzureDevOps(VSTS)-Security/Readme.md:353]: add group names

File: 09-AzureDevOps(VSTS)-Security/Readme.md (hunk 1 above)
Comment (abhaydaga) [reply 09-AzureDevOps(VSTS)-Security/Readme.md:270]: done

File: 09-AzureDevOps(VSTS)-Security/Readme.md (hunk 2 above)
Comment (abhaydaga) [reply 09-AzureDevOps(VSTS)-Security/Readme.md:271]: done

File: 09-AzureDevOps(VSTS)-Security/Readme.md (hunk 3 above)
Comment (abhaydaga) [reply 09-AzureDevOps(VSTS)-Security/Readme.md:353]: already mentioned in the notes below.

**Pull request merged**